packages/data-schema/src/SchemaProcessor.ts

```diff
@@ -974,15 +1008,19 @@ function processFields(
   fields: Record<string, any>,
   impliedFields: Record<string, any>,
   fieldLevelAuthRules: Record<string, string | null>,
+  getRefType: ReturnType<typeof getRefTypeForSchema>,
```

Comment:
I don't think we need to follow refs to generate types, in which case, I think this isn't needed.